deploy/ansible/gprofiler_playbook.yml

```diff
@@ -1,6 +1,17 @@
 #
-# Copyright (c) Granulate. All rights reserved.
-# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+# Copyright (C) 2023 Intel Corporation
```

Comment:
shouldn't it be 2024?

Reply:
Nope it should be 2023 which is the date Intel acquired us